snapshots/gapminder/2023-09-18/under_five_mortality.xlsx.dvc

```diff
@@ -0,0 +1,30 @@
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Child Mortality Rate (Gapminder)
+    description: |-
+      Gapminder’s historic estimates compiled by Klara Johansson and Mattias Lindgren.
+    date_published: 2015-01-01
+    version_producer: v7
+    title_snapshot: Child Mortality Rate (Gapminder)
```

Comment:
If you want to specify that you are extracting an old version of the data product, I suppose you could say `Child mortality rate under age five v7`.